11ty/definitions/nl_NL/biromantic.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Biromantic
+slug: biromantic
+speech: adj
+defined: true
+reading:
+  - text: Sorting Out Romantic vs Sexual Feelings
+    href: http://teenhealthsource.com/blog/faq-sorting-out-romantic-vs-sexual-feelings/
+  - text: Being Bi On The Asexual Spectrum
+    href: https://www.curvemag.com/Lifestyle/Being-Bi-On-The-Asexual-Spectrum-2141/
+  - text: Here’s What It Means When Your Romantic and Sexual Orientations Are Different
+    href: https://everydayfeminism.com/2016/07/cross-orientation-101/
+---
+
+Romantisch aangetrokken zijn tot meer dan één geslacht of sekse.
+
+## Noot
+
+Biromantic impliceert geen seksuele aantrekkingskracht. Een biromantic persoon kan aseksueel, homoseksueel, heteroseksueel, biseksueel, pan seksueel etc. zijn.
+Biromantic sluit romantisch aangetrokken zijn tot non-binaire of transgender personen niet uit.
```

Comment:
Slight grammar typo where I think the correct usage would be "transgenderpersoon" but then the sentence gets clunky, so this is probably fine. If not, perhaps reword to "non-binaire individuën en mensen die transgender zijn" or something?